client/src/mdn.scss

```diff
@@ -146,6 +146,10 @@ div.content {
     padding: 6px;
     border-radius: 3px;
   }
+
+  .link-list {
+    h2 { border-bottom: 1px solid #eaecef}
```

Comment:
Strange that Prettier didn't complain about the missing `;`. Prettier supports SCSS. Why did it not catch this??

Reply:
Yeah definitely pretty odd it didn't catch that..regardless i updated it.